dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
+        "label": "Central Alabama VA Medical Center-Montgomery"
       },
       {
         "value": "679",
-        "label": "TUSCALOOSA VA MEDICAL CENTER"
+        "label": "Tuscaloosa VA Medical Center"
       }
     ],
     "OR": [
       {
         "value": "653BY",
-        "label": "EUGENE, OR CBOC"
+        "label": "Eugene VA Clinic"
       },
       {
         "value": "653GA",
-        "label": "NORTH BEND VA CLINIC"
+        "label": "North Bend VA Clinic"
       },
       {
         "value": "653GB",
-        "label": "BROOKINGS VA CLINIC"
+        "label": "Brookings VA Clinic"
       },
       {
         "value": "648GA",
-        "label": "BEND, OR CBOC"
+        "label": "Bend VA Clinic"
       },
       {
         "value": "648GB",
-        "label": "SALEM VA CLINIC"
+        "label": "Salem VA Clinic"
       },
       {
         "value": "648GD",
-        "label": "NORTH COAST VA CLINIC"
+        "label": "North Coast VA Clinic"
       },
       {
         "value": "692GA",
-        "label": "KLAMATH FALLS VA CLINIC"
+        "label": "Klamath Falls VA Clinic"
       },
       {
         "value": "648GF",
-        "label": "HILLSBORO VA CLINIC"
+        "label": "Hillsboro VA Clinic"
       },
       {
         "value": "692GB",
-        "label": "GRANTS PASS CBOC"
+        "label": "Grants Pass VA Clinic"
       },
       {
         "value": "648GG",
-        "label": "WEST LINN CBOC"
+        "label": "West Linn VA Clinic"
       },
       {
         "value": "648GE",
-        "label": "EAST PORTLAND VA CLINIC"
+        "label": "Fairview VA Clinic"
       },
       {
         "value": "687GC",
-        "label": "LA GRANDE CBOC"
+        "label": "La Grande VA Clinic"
       },
       {
         "value": "648GH",
-        "label": "NEWPORT VA CLINIC"
+        "label": "Newport VA Clinic"
       },
       {
         "value": "531GH",
-        "label": "BURNS VA CLINIC"
+        "label": "Eastern Oregon VA Clinic"
       },
       {
         "value": "648GI",
-        "label": "PORTLAND VA CLINIC"
+        "label": "Portland VA Clinic"
       },
       {
         "value": "648GJ",
-        "label": "LOREN R KAUFMAN VA CLINIC"
+        "label": "Loren R. Kaufman VA Clinic"
       },
       {
         "value": "653QA",
-        "label": "DOWNTOWN EUGENE VA CLINIC"
+        "label": "Downtown Eugene VA Clinic"
       },
       {
         "value": "648",
-        "label": "PORTLAND VA MEDICAL CENTER"
+        "label": "Portland VA Medical Center"
       },
       {
         "value": "653",
-        "label": "ROSEBURG VA MEDICAL CENTER"
+        "label": "Roseburg VA Medical Center"
       },
       {
         "value": "692",
-        "label": "WHITE CITY VA MEDICAL CENTER"
+        "label": "White City VA Medical Center"
       }
     ],
     "OK": [
       {
         "value": "623BY",
-        "label": "TULSA, OK CBOC"
+        "label": "Ernest Childers Department of Veterans Affairs Outpatient Clinic"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.

Reply:
The city is TULSA, OK. So this checks out.